topopy/MorseSmaleComplex.py

```diff
@@ -238,6 +238,11 @@ def Build(self, X, Y, w=None, names=None, edges=None):
                 del partitions[min_max_pair]
 
         self.base_partitions = partitions
+
+        if(self.ValidCheck() >2 ):
```

Comment:
I think since it is technically valid to have more than one connected component, we can add a flag that will raise this warning only if the flag is set.

I will make the adjustment here.

Reply:
Sounds good